CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - `--max-function-trace-depth` allowing to specify maximum depth of the function tree in function level profiling
+- `--split-generics` flag allowing to differentiate between non-inlined generics monomorphised with different types
```

Comment:
Why not inlined is mentioned here?

Reply:
Because this flag doesn't work for inlined function - I cannot figure out the type it was monomorphised with from my mappings, I would have to change them which is not sth of high prio